### PR TITLE
Added gender in user.js, added routes for phone number, age and gende…

### PR DESCRIPTION
**Pull request by Yusuf-AH**
…r. Added for

Added gender and status in user.js, added routes for phone number, age and gender. Added for

### CODE REVIEW

File: src/controller/userController.js [hunk 1]
```diff
@@ -50,7 +50,38 @@ exports.setUserLastname = async (req, res) => {
     res.status(200).send(user);
 }
 
+<<<<<<< Updated upstream
```
Comment (goodhands) [inline src/controller/userController.js:53]: Wetin be this? 👀 Please update.

File: src/controller/userController.js [hunk 2]
```diff
@@ -50,7 +50,38 @@ exports.setUserLastname = async (req, res) => {
...
+
+exports.getgender = async (req, res) => {
+    const user = await User.findOne({ _id: req.params.id });
+    res.status(200).send(user.gender);
+}
+
+exports.setgender = async (req, res) => {
+    const user = await User.findOneAndUpdate({ _id: req.params.id }, {gender : req.body.gender});
+    res.status(200).send(user);
+>>>>>>> Stashed changes
```
Comment (goodhands) [inline src/controller/userController.js:87]: You have to remove these git comments

File: src/models/user.js [hunk 3]
```diff
@@ -38,6 +38,18 @@ const User = mongoose.model('User', {
     age: {
         type: Number,
         default: 0
+    },
+    gender: {
+        type: String,
+        enum: ['male', 'female']
```
Comment (goodhands) [inline src/models/user.js:44]: Nice one

File: src/routes/index.js [hunk 4]
```diff
@@ -28,7 +28,26 @@ router.put('/users/:id/firstname', catchErrors(userController.setUserFirstname))
 router.get('/users/:id/lastname', catchErrors(userController.getUserLastName));
 //Set Last Name
 router.put('/users/:id/lastname', catchErrors(userController.setUserLastname));
+<<<<<<< Updated upstream
 // GET Specific User Emai;
 router.get('/user/:id/email', catchErrors(userController.getUserEmail));
+=======
+
+//Get Phone number
+router.get('/users/:id/phoneNumber', catchErrors(userController.getphoneNumber));
```
Comment (goodhands) [inline src/routes/index.js:37]: `phone` is okay, no need for `phoneNumber`

Comment (Yusuf-AH) [reply src/routes/index.js:37]: I used it because the user property was "phoneNumber".

I'll change it.

**Pull request closed without merging**